database/auth.go

```diff
@@ -114,6 +114,55 @@ func (db *Db) storeVestingAccount(account exported.VestingAccount) (int, error)
 	return vestingAccountRowID, nil
 }
 
+func (db *Db) StoreBaseVestingAccountFromMsg(bva *vestingtypes.BaseVestingAccount) error {
+	stmt := `
+	INSERT INTO vesting_account (type, address, original_vesting, end_time) 
```

Comment:
This func never updates `start_time` value so it's always null in db. Since we can't get it from `BaseVestingAccount` type, can you try to pass the tx timestamp and store it as `start_time` value?